regression/pages/lms/login_lms.py

```diff
@@ -0,0 +1,38 @@
+from bok_choy.promise import EmptyPromise
+from edxapp_pages.lms.login import LoginPage
+from edxapp_pages.lms.dashboard import DashboardPage
+from regression.pages.lms import BASE_URL
+
+
+class LmsLogin(LoginPage):
+    """
+    This class is an extended class of LoginPage,
+    where we add methods that are different or not used in LoginPage
+    """
+    url = BASE_URL + '/login'
+
+    def is_browser_on_page(self):
+        return self.q(css='.action.action-primary.action-update.js-login.login-button').present
+
+    def provide_info(self, email, password):
+        """
+        Fill in login info
+        'Username' and 'Password' are the user's credentials
+        """
+        EmptyPromise(self.q(css='input#login-email').is_present, "Click ready").fulfill()
```

Comment:
To avoid repeating `input#login-email`, you should set that in a variable at the top of the method, then reference the variable. Same goes for `login-password`.